data/integration_tests/percolator/IntegrationTest_Percolator_SpeedGraph.py

```diff
@@ -0,0 +1,167 @@
+# Marcus Andersson - Percolator Project
+# Takes data from IntegrationTest_Percolator_SpeedGraphData.py and outputs graph images.
+# Input: Any number of data files, but at least 1.
+
+pathToOutputData = "@pathToOutputData@"
+pathToTestScripts = "@pathToTestScripts@"
+
+import sys
+
+import pathlib
+import datetime
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import ast 
+from matplotlib.cm import get_cmap
+import matplotlib.lines as mlines
+from matplotlib.ticker import FormatStrFormatter
+
+
+markers = ["o","v","^","<",">","1","s","p","P","*","H","x","d","|","+"]
+outputFilename = "outGraph"
+
+def getMinExponent(xStepValue):
```

Comment:
Maybe add a comment explaining this function? On a first reading, I would have expected the exponent variable to be added to on each iteration, not multiplied, for example. A comment (and/or different variable names) would help here.

(Comments in general explaining what functions do, that aren't immediately obvious to a first-time reader, would be nice.)

Reply:
Done.